app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -255,16 +258,23 @@ private function getOrCreateUser($record, $values)
                 'source' => env('NORTHSTAR_CLIENT_ID'),
             ];
 
-            if ($record['Phone']) {
-                $userData['sms_status'] = $this->transformSmsStatus($record['Opt-in to Partner SMS/robocall']);
+            $recordEmailOptIn = $record['Opt-in to Partner email?'];
+            if ($recordEmailOptIn) {
```

Comment:
do we need to check if this `isset` and `!empty`?

Reply:
I've added the check for `isset`, but looking into [the `empty` PHP docs](http://php.net/manual/en/function.empty.php) -- I'm not sure we'd want to use it per adding the `str_to_boolean` helper in this PR.

Per the docs:
> The following values are considered to be empty:
> - "" (an empty string)
> - 0 (0 as an integer)
> - 0.0 (0 as a float)
> - "0" (0 as a string)
> - NULL
> - FALSE
> - array() (an empty array)

Per our `str_to_boolean`, we're checking for more than just 'no' values.  If there is a 0 set in our column, we'll interpet it as a `false` value to save (vs not saving it at all)

Reply:
Yeah, I'd be wary of over-using `isset()` or `empty()` unless you specifically need the functionality they're providing (checking whether a variable is defined, like a nested array reference / checking whether a variable is defined & truthy).

Since they swallow up so many "false" conditions, they could hide sneaky bugs.

Reply:
The initial code was simply checking for `if ($record['Phone'])` vs isset, empty -- do we even need the `isset`? I was also looking at `array_key_exists`...but it seems unlikely we'd ever receive a NULL value via importing the csv

Reply:
Yeah, I don't think we need an `isset` here. 👍